src/backend/gl/src/factory.rs

```diff
@@ -88,6 +88,11 @@ impl Factory {
         }
     }
 
+    /// Get the OpenGL-specific driver information
+    pub fn get_info<'a>(&'a self) -> &'a Info {
```

Comment:
That could also be done, indeed. But I still would stick with the current solution because `Share` (containing `Info`) is wrapped in a `Rc` and we need `Info` anyways for the CommandQueue later, either stored directly inside the struct or also in `Share` if we want to expose multiple queues for the GL backend.